### PR TITLE
EVG-7823 add a commit queue message

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: operations/commit_queue.go [hunk 1]
```diff
@@ -385,6 +399,7 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 	}
 	grip.Infof("Project: %s\n", projectID)
 	grip.Infof("Type of queue: %s\n", projectRef.CommitQueue.PatchType)
+	grip.Infof("Message: %s\n", projectRef.CommitQueue.Message)
```
Comment (ybrill) [inline operations/commit_queue.go:402]: won't this print the message a second time? (given it's already printed [here](https://github.com/evergreen-ci/evergreen/pull/4048/files?diff=split&w=1#diff-7f9dabe9369339d415247be60a5d1b45R77))

Comment (john-m-liu) [reply operations/commit_queue.go:401]: ah yes, forgot about that. Removed it in the other place

**Review by ybrill — APPROVED** [review]

**Comment by ybrill** [comment]
It just occurred to me that we should increment the CLI version string

File: config.go [hunk 2]
```diff
@@ -31,10 +31,10 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2020-09-29"
+	ClientVersion = "2020-09-30"
 
 	// Agent version to control agent rollover.
-	AgentVersion = "2020-09-21"
+	AgentVersion = "2020-09-30"
```
Comment (john-m-liu) [inline config.go:37]: https://github.com/evergreen-ci/evergreen/commit/ac8229973df2a24ed1291b48517aff4d5d82b723 should have incremented the agent version but didn't, so I'm slipping it in here

**Pull request merged**